### PR TITLE
2 files for move cal

### PR DESCRIPTION
**Pull request by chaimmendelson**

### CODE REVIEW

**Review by amos992005 — APPROVED** [review]

File: src/possible_moves.c [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+
+int func(){
+    board start_board;
+    char pieces[13] = {'E','N','R','Q','K','P','B','n','r','q','k','p','b'};
+    char initial_board[32] = {white_rook << 4|white_knight,white_bishop << 4|white_queen,white_king<<4|
+    white_bishop,white_knight<<4|white_rook,white_pawn << 4|white_pawn,white_pawn<<4|white_pawn,white_pawn<<4|
+    white_pawn,white_pawn<<4|white_pawn,empty,empty,empty,empty,empty,empty,empty,empty,empty,empty,empty,empty
+    ,empty,empty,empty,empty,black_pawn << 4|black_pawn,black_pawn<<4|black_pawn,black_pawn<<4|black_pawn,black_pawn<<4|
+    black_pawn,black_rook << 4|black_knight,black_bishop << 4|black_queen,black_king<<4|black_bishop,black_knight<<4|black_rook};
+    int i = 0;
```
Comment (elchairoy) [inline src/possible_moves.c:11]: add a space

File: src/possible_moves.c [hunk 2]
```diff
@@ -0,0 +1,17 @@
+#include "possible_moves.h"
+
+int func(){
+    board start_board;
+    char pieces[13] = {'E','N','R','Q','K','P','B','n','r','q','k','p','b'};
+    char initial_board[32] = {white_rook << 4|white_knight,white_bishop << 4|white_queen,white_king<<4|
+    white_bishop,white_knight<<4|white_rook,white_pawn << 4|white_pawn,white_pawn<<4|white_pawn,white_pawn<<4|
+    white_pawn,white_pawn<<4|white_pawn,empty,empty,empty,empty,empty,empty,empty,empty,empty,empty,empty,empty
+    ,empty,empty,empty,empty,black_pawn << 4|black_pawn,black_pawn<<4|black_pawn,black_pawn<<4|black_pawn,black_pawn<<4|
+    black_pawn,black_rook << 4|black_knight,black_bishop << 4|black_queen,black_king<<4|black_bishop,black_knight<<4|black_rook};
```
Comment (elchairoy) [inline src/possible_moves.c:10]: add a space

File: src/possible_moves.c [hunk 3]
```diff
@@ -0,0 +1,17 @@
+#include "possible_moves.h"
+
+int func(){
```
Comment (elchairoy) [inline src/possible_moves.c:3]: rename the function

File: src/possible_moves.c [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+    char initial_board[32] = {white_rook << 4|white_knight,white_bishop << 4|white_queen,white_king<<4|
+    white_bishop,white_knight<<4|white_rook,white_pawn << 4|white_pawn,white_pawn<<4|white_pawn,white_pawn<<4|
+    white_pawn,white_pawn<<4|white_pawn,empty,empty,empty,empty,empty,empty,empty,empty,empty,empty,empty,empty
+    ,empty,empty,empty,empty,black_pawn << 4|black_pawn,black_pawn<<4|black_pawn,black_pawn<<4|black_pawn,black_pawn<<4|
+    black_pawn,black_rook << 4|black_knight,black_bishop << 4|black_queen,black_king<<4|black_bishop,black_knight<<4|black_rook};
+    int i = 0;
+    for(i;i<32;i++)
+        start_board.squares[i] = initial_board[i];
+    int piece = get_piece_in_square(&start_board, 1);
+    printf("hello%c", pieces[piece]);
```
Comment (elchairoy) [inline src/possible_moves.c:15]: remove the hello

File: src/possible_moves.c [hunk 5]
```diff
@@ -0,0 +1,17 @@
...
+int func(){
+    board start_board;
+    char pieces[13] = {'E','N','R','Q','K','P','B','n','r','q','k','p','b'};
+    char initial_board[32] = {white_rook << 4|white_knight,white_bishop << 4|white_queen,white_king<<4|
+    white_bishop,white_knight<<4|white_rook,white_pawn << 4|white_pawn,white_pawn<<4|white_pawn,white_pawn<<4|
+    white_pawn,white_pawn<<4|white_pawn,empty,empty,empty,empty,empty,empty,empty,empty,empty,empty,empty,empty
+    ,empty,empty,empty,empty,black_pawn << 4|black_pawn,black_pawn<<4|black_pawn,black_pawn<<4|black_pawn,black_pawn<<4|
+    black_pawn,black_rook << 4|black_knight,black_bishop << 4|black_queen,black_king<<4|black_bishop,black_knight<<4|black_rook};
+    int i = 0;
+    for(i;i<32;i++)
```
Comment (elchairoy) [inline src/possible_moves.c:12]: add spaces

**Pull request closed without merging**